gulpfile.js

```diff
@@ -21,6 +21,7 @@ var gulp = require('gulp'),
     connectModRewrite = require('connect-modrewrite'),
     ngConfig = require('gulp-ng-config'),
     prettyError = require('gulp-prettyerror'),
+    path = require('path'),
```

Comment:
Is this dependent on npm? If yes, can you also add the package.json file, I think you forgot to add it.